client/src/pages/Login.js

```diff
@@ -0,0 +1,166 @@
+    fontFamily: theme.typography.fontFamily
+  },
+  footer: {
+    fontFamily: theme.typography.fontFamily,
+    padding: "5%",
+    textAlign: "center"
+  },
+  logo: {
+    marginTop: "10vh",
+    marginLeft: "auto",
+    marginRight: "auto",
+    marginBottom: "5vh"
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  },
+  button: {
+    background: theme.palette.primary.button,
+    color: "white",
+    padding: "15px 50px 15px 50px",
+    marginTop: "15%",
+    marginBottom: "10%"
+  },
+  link: {
+    marginLeft: 3,
+    color: theme.palette.primary.main,
+    textDecoration: "none"
+  },
+  paper: {
+    width: "80%",
+    margin: "auto"
+  },
+  formInput: {
+    marginTop: "20%",
+    width: "90%"
+  },
+  formLabel: {
+    marginBottom: "-10px",
+    textAlign: "center"
+  }
+}));
+
+const LogInPage = () => {
+  const classes = useStyles();
+  const [email, setEmail] = useState(null);
+  //welcomeMsg is true if use has entered an email and pressed continue button
+  const [welcomeMsg, showWelcome] = useState(false);
+
+  const handleClick = event => {
+    event.preventDefault();
+    //cehck if user has entered an email
+    if (email) {
+      showWelcome(true);
+    }
+  };
+
+  const handleChange = event => {
+    setEmail(event.target.value);
+    console.log(email);
+  };
+
+  return (
+    <Container
+      maxWidth="sm"
+      classes={{
+        root: classes.root
+      }}
+    >
+      <img src={logo} alt="company logo" className={classes.logo} />
+      <Paper elevation={5} className={classes.paper}>
+        <form className={classes.formMain} onSubmit={handleClick}>
+          <Typography variant="h5">
+            {welcomeMsg ? (
+              <span
+                style={{
+                  display: "flex",
+                  flexDirection: "column",
+                  textAlign: "center"
+                }}
+              >
+                <span>{`Welcome back,`}</span>
+                <span>{email}</span>
+              </span>
+            ) : (
+              `Log into your account`
+            )}
+          </Typography>
+          {welcomeMsg ? null : (
+            <div className={classes.formInput}>
+              <Typography variant="h6" className={classes.formLabel}>
+                Enter your E-mail to get started:
+              </Typography>
+              <TextField
+                variant="outlined"
+                margin="normal"
+                required
+                fullWidth
+                id="email"
+                name="email"
+                autoComplete="email"
+                placeholder="E-mail address"
+                autoFocus
+                style={{ textAlign: "center" }}
+                inputProps={{ min: 0, style: { textAlign: "center" } }}
+                onChange={handleChange}
+              />
+            </div>
+          )}
+          <Button
+            size="large"
+            className={classes.button}
+            onClick={handleClick}
```

Comment:
Instead of having an onClick, it would be better for accessibility to make this type="submit" and it will use the onSubmit for the form